Marlin/language_en.h

```diff
@@ -602,6 +602,45 @@
 #ifndef MSG_FILAMENTCHANGE
   #define MSG_FILAMENTCHANGE                  _UxGT("Change filament")
 #endif
+#ifndef MSG_FILAMENTLOADUNLOAD
+  #define MSG_FILAMENTLOADUNLOAD              _UxGT("Filament Un/Load")
+#endif
```

Comment:
This menu could also be called Change Filament — or these options could simply be included in the existing Change Filament menu, since load and unload are each just half of a filament change.